README.md

```diff
@@ -66,59 +56,24 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-### Third party CI systems
```

Comment:
I would personally keep this old one here with the subtitle Third party CI systems. The check suite seems to have been intended for that purpose. So let's address workflow run first.

Reply:
I'm not sure, I think it was `workflow-run` that was intended for 3rd party CI systems, but now it has become default. I left it open for the consumer to decide which trigger to run, without suggesting the use case.

Reply:
But it makes more sense to have `workflow_run` by default I agree.

Reply:
But it makes more sense to have `workflow_run` by default I agree. 00b19b0

Reply:
@JrSchild I am going to merge this one as #844 depends on it. The second one I need to merge for all Dependabot PRs that will be rebased to succeed. If you have comments still I'll create another PR for it.